<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workertab</class>
 <widget class="QWidget" name="workertab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pb_new_pr">
     <property name="text">
      <string>Новый</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="leSearchPr">
     <property name="placeholderText">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbReport">
     <property name="text">
      <string>Отчет</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pbSearch">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_post"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="le_storage_id"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="le_salary"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="le_worker_fio"/>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pbDeletePr">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ФИО работника</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QPushButton" name="pbEditPr">
        <property name="text">
         <string>Редактировать</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Зарплата</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Код магазина</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Должность</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cb_storage"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Магазин</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pbUpdateColumn">
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="4">
    <widget class="QTableWidget" name="twCustomers"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
